haiku-apps/streamradio/streamradio-0.0.5_1.recipe

```diff
@@ -5,10 +5,10 @@ HOMEPAGE="https://github.com/fishpond-haiku/Haiku-Radio"
 COPYRIGHT="2017 Kai Niessen"
 LICENSE="MIT"
 REVISION="4"
```

Comment:
Please reset the revision.

Reply:
Done. Sorry, I thought that field was referred to the license revision. My bad.